esileapclient/tests/functional/README.md

```diff
@@ -0,0 +1,50 @@
+# esi-leap Client Functional Tests
+
+These tests are designed to ensure the esi-leap services and the esi-leap CLI client function as expected when interacting with each other.
```

Comment:
Nit: "when interacting with each other" feels extraneous.

Reply:
True. I think my reasoning behind this one was that the functional tests make sure the esi-leap workflow as a whole (the combination of the client + services) work well as opposed to the two working fine independently but having issues when interfacing with each other. I think dropping the "when interacting..." part sounds better though.